From: Chirayu Desai <chirayudesai1@gmail.com>
Date: Fri, 3 Jun 2022 02:56:06 +0530
Subject: Update default popular sites

* This can be done in a better way I think,
  but for now it should do
* We can add more sites later

Change-Id: I2fb4f7fb1fcb14cbf9749809673fc2042b627223
---
 .../resources/default_popular_sites.json      | 44 +++++--------------
 1 file changed, 12 insertions(+), 32 deletions(-)

diff --git a/components/ntp_tiles/resources/default_popular_sites.json b/components/ntp_tiles/resources/default_popular_sites.json
--- a/components/ntp_tiles/resources/default_popular_sites.json
+++ b/components/ntp_tiles/resources/default_popular_sites.json
@@ -1,42 +1,22 @@
 [
     {
-        "large_icon_url": "https://www.chromium.org/_/rsrc/1484148948583/apple-touch-icon.png",
-        "title": "The Chromium Projects `",
-        "url": "https://www.chromium.org/"
+        "large_icon_url": "https://calyxos.org/assets/images/favicon.png",
+        "title": "Calyx Institute",
+        "url": "https://calyxinstitute.org/"
     },
     {
-        "favicon_url": "https://codereview.chromium.org/static/favicon.ico",
-        "title": "Chromium Code Reviews",
-        "url": "https://codereview.chromium.org/"
+        "large_icon_url": "https://calyxos.org/assets/images/favicon.png",
+        "title": "CalyxOS",
+        "url": "https://calyxos.org/"
     },
     {
-        "large_icon_url": "https://www.chromium.org/_/rsrc/1484148948583/apple-touch-icon.png",
-        "title": "Git Repositories on chromium",
-        "url": "https://chromium.googlesource.com/"
+        "large_icon_url": "https://gitlab.com/assets/apple-touch-icon-b049d4bc0dd9626f31db825d61880737befc7835982586d015bded10b4435460.png",
+        "title": "CalyxOS Issue Tracker",
+        "url": "https://gitlab.com/CalyxOS/calyxos/-/issues"
     },
     {
-        "large_icon_url": "https://developers.google.com/_static/9bce7b6017/images/touch-icon.png",
-        "title": "Google Open Source Programs Office",
-        "url": "https://developers.google.com/open-source/"
-    },
-    {
-        "large_icon_url": "https://freenode.net/static/img/logos/coloured-alphabg-sq-120.png",
-        "title": "freenode",
-        "url": "https://freenode.net/"
-    },
-    {
-        "large_icon_url": "https://assets-cdn.github.com/pinned-octocat.svg",
-        "title": "Github Page of the Chromium Project",
-        "url": "https://github.com/chromium/"
-    },
-    {
-        "large_icon_url": "https://cdn2.iconfinder.com/data/icons/minimalism/512/Chromium.png",
-        "title": "Chrome Flags",
-        "url": "chrome://flags"
-    },
-    {
-        "large_icon_url": "https://cdn2.iconfinder.com/data/icons/minimalism/512/Chromium.png",
-        "title": "Chrome Version",
-        "url": "chrome://version/"
+        "large_icon_url": "https://www.bromite.org/android-icon-192x192.png",
+        "title": "Bromite",
+        "url": "https://www.bromite.org/"
     }
 ]
-- 
2.40.1

